contracts/test/Dependencies.sol

```diff
@@ -0,0 +1,9 @@
+pragma solidity 0.5.0;
+
+// hack for truffle, so we have artifact for dependencies contracts for migrations and tests
```

Comment:
I personally don't see what the point of this is here. I've never had to do this.

Reply:
we deploy external Contracts like VerifierRegistry, HumanStandardToken, etc. during the tests. truffle needs to generate ABI files so we can work with the contracts.